disttask/framework/scheduler/manager.go

```diff
@@ -373,11 +374,14 @@ func (m *Manager) removeHandlingTask(id int64) {
 	delete(m.mu.handlingTasks, id)
 }
 
-// onError handles the error.
-func (m *Manager) onError(err error) {
-	if err == nil {
-		return
-	}
-
+func (m *Manager) logErr(err error) {
 	logutil.Logger(m.logCtx).Error("task manager error", zap.Error(err))
 }
+
+func (m *Manager) logErrAndPersist(err error, taskID int64) {
+	m.logErr(err)
+	err1 := m.taskTable.UpdateErrorToSubtask(m.id, taskID, err)
```

Comment:
When update to subtask failed, what will happen?
We can use backoffer.

Reply:
It will act like `logErr()` and keep retrying.